src/pages/_error.tsx

```diff
@@ -1,23 +1,98 @@
-import React from "react";
+import React, { useContext, useEffect } from "react";
 import { NextPageContext } from "next";
 import Layout from "../components/Layout/Layout";
 import Link from "../components/Link/Link";
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Image,
+  Text,
+} from "@nypl/design-system-react-components";
+import { FeedbackContext } from "../context/FeedbackContext";
+
+const ERROR_PERSISTS = " if the error persists.";
+
+const getNotificationText = (statusCode) => {
+  return `You are asking for help or information about a page error (error code ${statusCode})`;
+};
+
+const errorMap = {
+  500: {
+    overline: "error 500",
+    heading: "Something went wrong on our end.",
+    subText: "We encountered an error while trying to load the page. ",
+    tryText: "Try refreshing the page or ",
+  },
+  404: {
+    overline: "error 404",
+    heading: "We couldn't find that page.",
+    subText:
+      "The page you were looking for doesn't exist or may have moved elsewhere. ",
+    tryText: "Try a different URL or ",
+  },
+  400: {
+    overline: "error 500",
+    heading: "There was an unexpected error.",
+    subText: "We couldn't process your request at this time. ",
+    tryText: "Try refreshing the page or ",
+  },
```

Comment:
Hmm 400 means that the request input was invalid. Should we reflect that accordingly in the text?